07-space.Rmd

```diff
@@ -383,10 +383,9 @@ crashes_iow = readRDS(url(u))
 ```
 
 1. Plot the roads with the crashes overlaid.
-2. Create a buffer around the roads with a distance of 200 m.
+2. Create a buffer around the roads with a distance of 200m.
```

Comment:
Really? I disagree, from an engineering writing perspective it's 200 m: https://english.stackexchange.com/questions/351906/use-of-space-before-a-unit

Reply:
Whichever you prefer I think